docs/src/pages/components/ProgressBar.svx

```diff
@@ -21,6 +21,18 @@ Specify a `value` for the progress bar to be determinate.
 
 <ProgressBar value={40} labelText="Upload status" helperText="40 MB of 100 MB" />
 
+## Finished status
+
+Specify `status="finished"` for the progress bar.
+
+<ProgressBar value={100} status="finished" labelText="Upload status" helperText="100 MB of 100 MB" />
```

Comment:
I recommend updating these labels:

- `labelText="Upload file"`
- `helperText="Upload complete"`

Reference: https://carbondesignsystem.com/components/progress-bar/accessibility#provide-status-for-indeterminate-and-errored-progress-bars